sherpa/cpp_api/online_stream.h

```diff
@@ -40,7 +41,8 @@ class OnlineStream {
    *                            If it is set to -1, we keep all feature frames
    *                            computed so far.
    */
-  OnlineStream(float sampling_rate, int32_t feature_dim,
+  OnlineStream(const EndpointConfig endpoint_config,
```

Comment:
If you are not using a pointer or a reference for `EndpointConfig`, then you need to use `include` to include the header. Just use forward declaration is not enough.

Reply:
Just fixed it. It's a bit strange that there is no error when compiling on the MAC OS.